code/modules/species/station/xenomorph_hybrids/hybrid_abilities.dm

```diff
@@ -9,3 +9,30 @@
 		if(O)
 			O.color = "#422649"
 	return
+
+/datum/ability/species/xenomorph_hybrid
+	action_icon = 'icons/screen/actions/xenomorph.dmi'
+
+/datum/ability/species/xenomorph_hybrid/regenerate
+	name = "Rest and regenerate"
+	desc = "Lie down and regenerate your health"
+	action_state = "regenerate"
+	windup = 0 SECOND
+	interact_type = ABILITY_INTERACT_TRIGGER
+	always_bind = TRUE
+	ability_check_flags = ABILITY_CHECK_RESTING
+	mobility_check_flags = MOBILITY_IS_CONSCIOUS
+
+/datum/ability/species/xenomorph_hybrid/regenerate/on_trigger()
+	. = ..()
+	var/mob/living/carbon/human/O = owner
+	if(istype(O))
+		to_chat(O, SPAN_NOTICEALIEN("We begin to mend our wounds."))
+		O.active_regen = TRUE
+
+
+
+
+
+
+
```

Comment:
slightly excessive number of end lines

Reply:
I place those when working near the end of a file since VSC adds an empty newline when you write into the lastline, and it usually tears what ever I am typing apart